[FW][FIX] account_payment_group: remove kanban from mode

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/account-payment#456

rov-adhoc committed:
[FIX] account_payment_group: remove kanban from mode

X-original-commit: f02ca6f24435ebd3d185a7c0fb5b3226c0caeb8e

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/457).

fw-bot-adhoc commented:
@rov-adhoc @jjscarafia cherrypicking of pull request ingadhoc/account-payment#456 failed.

stdout:
```
CONFLICT (modify/delete): account_payment_group/views/account_payment_group_view.xml deleted in HEAD and modified in 2de83318 ([FIX] account_payment_group: remove kanban from mode). Version 2de83318 ([FIX] account_payment_group: remove kanban from mode) of account_payment_group/views/account_payment_group_view.xml left in tree.
CONFLICT (modify/delete): account_payment_group/__manifest__.py deleted in HEAD and modified in 2de83318 ([FIX] account_payment_group: remove kanban from mode). Version 2de83318 ([FIX] account_payment_group: remove kanban from mode) of account_payment_group/__manifest__.py left in tree.

```

stderr:
```
19:22:35.214365 git.c:444               trace: built-in: git cherry-pick 2de833183046e908579d1df375cec9ad46f4081c
error: could not apply 2de83318... [FIX] account_payment_group: remove kanban from mode
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
